lib/util/validator.ts

```diff
@@ -33,4 +33,13 @@ export class FieldValidator {
   public static readonly tradeType = Joi.string().valid('EXACT_INPUT', 'EXACT_OUTPUT');
 
   public static readonly uuid = Joi.string().guid({ version: 'uuidv4' });
+
+  // A Raw Signature is a common Signature format where the r, s and v
+  //   are concatenated into a 65 byte(130 nibble) DataHexString
+  public static readonly rawSignature = Joi.string().custom((value: string, helpers: CustomHelpers<string>) => {
```

Comment:
nit: i thought sig length details were handled under the hood by ethers?

Reply:
they are, but that wouldn't throw a nice early validation result - we do this for several things, put a coarse validation check in joi so it catches obvious errors early